haskell-tests/Types/PayloadSerializationSpec.hs

```diff
@@ -278,21 +53,63 @@ groupIntoSize s =
 
 checkPayload :: SProtocolVersion pv -> Payload -> Property
 checkPayload spv e = let bs = S.runPut $ putPayload e
-                     in case S.runGet (getPayload spv (fromIntegral (BS.length bs))) bs of
-                          Left err -> counterexample err False
-                          Right e' -> label (groupIntoSize (fromIntegral (BS.length bs))) $ e === e'
+                 in case S.runGet (getPayload spv (fromIntegral (BS.length bs))) bs of
+                      Left err -> counterexample err False
+                      Right e' -> label (groupIntoSize (fromIntegral (BS.length bs))) $ e === e'
+
+modifyPayloadBitmap :: (Word16 -> Word16) -> BS.ByteString -> BS.ByteString
```

Comment:
Provide documentation. The name is very generic and does not inform the reader about what this does.